gnd/src/main/res/layout/basemap_selector_frag.xml

```diff
@@ -0,0 +1,32 @@
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <data>
+
```

Comment:
Nit: Extra line

Reply:
Removed. Weirdly enough, when I run the java formatter it insists on adding this line back in.